glslc/README.asciidoc

```diff
@@ -208,12 +208,9 @@ empty value.
 
 ==== `-I`
 
-`-I` adds the specified directory to the search path for include files.
-
-Only `""` style includes are permitted. When searching for included files, first
-the directory of the current file is attempted. If this fails each directory
-supplied via `-I` on the command line is searched in the given order for the
-file.
+`-Idirectory` adds the specified directory to the search path for include files.
```

Comment:
By `-Idirectory`, it seems to me that `-I directory` is not valid. I'd suggest just using `-I` here.